### ISSUE TITLE
Docs - We are missing resources in the docs

### ISSUE THREAD

**Issue by DavidGOrtega** [iterative/terraform-provider-iterative#345]
We do no have docs about `machine` and `runner` resources

**Comment by 0x2b3bfa0** [comment]
🤔 Should we?

**Comment by casperdcl** [comment]
I think we can put docs about them in https://github.com/iterative/terraform-provider-iterative/wiki for now

**Comment by 0x2b3bfa0** [comment]
After #461 this looks like <kbd>won't fix</kbd> or, at least, in dire need of reformulation
